### DISCUSSION TITLE
the backup file only has obj last weight, it doesn't do a backup every 1000 iterations, why is it like that?

### DISCUSSION THREAD

**Discussion by kemaltf** [AlexeyAB/darknet#8406] [Q&A] [answered]
I did training on yolov4 on google colabs. and I only get last weight no backup files per 1000 iterations. How can be like that?

**Comment by stephanecharette** [comment, marked as answer]
If max batches is <= 10K, then it saves every 1K.
If max batches is > 10K, then it only saves every 10K.

**Reply by kemaltf** [reply]
thank you for your answer!
